pkg/scheduler/core/generic_scheduler.go

```diff
@@ -458,7 +458,7 @@ func (g *genericScheduler) findNodesThatFit(state *framework.CycleState, pod *v1
 	failedPredicateMap := FailedPredicateMap{}
 	filteredNodesStatuses := framework.NodeToStatusMap{}
 
-	if len(g.predicates) == 0 {
+	if len(g.predicates) == 0 && !g.framework.HasFilterPlugins() {
```

Comment:
Will there be any default filter plugins if not configured?

Reply:
I am not sure what you mean, but the else clause should be executed if there is at least one filter or predicate to evaluate.

Reply:
There will be default plugins configured, but how is this relevant to this line?

Reply:
Sorry, I checked the code again, it works.